### ISSUE TITLE
Default Transmission web UI unresponsive

### ISSUE THREAD

**Issue by allefeld** [haugene/docker-transmission-openvpn#2137]
### Is there a pinned issue for this?

- [X] I have read the pinned issues and could not find my issue

### Is there an existing or similar issue/discussion for this?

- [X] I have searched the existing issues
- [X] I have searched the existing discussions

### Is there any comment in the documentation for this?

- [X] I have read the documentation, especially the FAQ and Troubleshooting parts

### Is this related to a provider?

- [X] I have checked the provider repo for issues
- [X] My issue is NOT related to a provider

### Are you using the latest release?

- [X] I am using the latest release

### Have you tried using the dev branch latest?

- [ ] I have tried using dev branch

### Config used

I'm running the container like this

````
args=(
    # Docker network administration capability
    --cap-add=NET_ADMIN
    # Docker logging options
    --log-driver json-file
    --log-opt max-size=10m
    # Docker container name
    --name dto
    # where data are stored, must be linked to `/data` within container
    -v /home/ca/Store/transmission:/data
    # OpenVPN configuration for Private Internet Access
    -e OPENVPN_PROVIDER=PIA
    -e OPENVPN_CONFIG=netherlands,switzerland,sweden,france,liechtenstein,denmark,andorra,belgium,iceland,monaco,norway,spain,luxembourg
    -e OPENVPN_USERNAME=XXX
    -e OPENVPN_PASSWORD=XXX
    # make sure local network traffic is not routed through VPN
    -e LOCAL_NETWORK=192.168.0.0/16
    # owner and group of stored files
    -e PUID=1000
    -e PGID=1000
    # make Transmission web UI accessible from outside, as `localhost:9091`
    -p 9091:9091
    # run Tinyproxy and make it accessible from outside, as `localhost:8888`
    # used e.g. through `vgc`
    -e WEBPROXY_ENABLED=true
    -p 8888:8888
    )
docker run -d "${args[@]}" haugene/transmission-openvpn
````

### Current Behavior

I can access the Transmission Web UI, but all of the UI elements are unresponsive. Among other things, I can't add a torrent.

### Expected Behavior

Being able to use the default Transmission Web UI.

### How have you tried to solve the problem?

I can make it work by using an older version of the image, `haugene/transmission-openvpn:3.6`.

With the latest image version, I can use Transmission if I configure an alternative Web UI, e.g. `-e TRANSMISSION_WEB_UI=combustion`.

The problem seems to be specifically the combination of the latest image version with the standard Transmission Web UI.

### Log output

````
Starting container with revision: 20877f1b168b6ff27fc58aeef40756572c562d47
Creating TUN device /dev/net/tun
Using OpenVPN provider: PIA
Running with VPN_CONFIG_SOURCE auto
Provider PIA has a bundled setup script. Defaulting to internal config
Executing setup script for PIA
Downloading OpenVPN config bundle openvpn into temporary file /tmp/tmp.5Nf7O78xpE
Extract OpenVPN config bundle into PIA directory /etc/openvpn/pia
13 servers found in OPENVPN_CONFIG, iceland chosen randomly
Starting OpenVPN using config iceland.ovpn
Modifying /etc/openvpn/pia/iceland.ovpn for best behaviour in this container
Modification: Point auth-user-pass option to the username/password file
Modification: Change ca certificate path
Modification: Change ping options
Modification: Update/set resolv-retry to 15 seconds
Modification: Change tls-crypt keyfile path
Modification: Set output verbosity to 3
Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
Setting OpenVPN credentials...
adding route to local network 192.168.0.0/16 via 172.17.0.1 dev eth0
Sun Jan  2 22:16:45 2022 OpenVPN 2.4.7 x86_64-pc-linux-gnu [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [PKCS11] [MH/PKTINFO] [AEAD] built on Jul 19 2021
Sun Jan  2 22:16:45 2022 library versions: OpenSSL 1.1.1f  31 Mar 2020, LZO 2.10
Sun Jan  2 22:16:45 2022 NOTE: the current --script-security setting may allow this configuration to call user-defined scripts
Sun Jan  2 22:16:46 2022 TCP/UDP: Preserving recently used remote address: [AF_INET]45.133.193.41:1198
Sun Jan  2 22:16:46 2022 Socket Buffers: R=[212992->212992] S=[212992->212992]
Sun Jan  2 22:16:46 2022 UDP link local: (not bound)
Sun Jan  2 22:16:46 2022 UDP link remote: [AF_INET]45.133.193.41:1198
Sun Jan  2 22:16:46 2022 TLS: Initial packet from [AF_INET]45.133.193.41:1198, sid=f5f655c8 a51c5299
Sun Jan  2 22:16:46 2022 WARNING: this configuration may cache passwords in memory -- use the auth-nocache option to prevent this
Sun Jan  2 22:16:46 2022 VERIFY OK: depth=1, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=Private Internet Access, name=Private Internet Access, emailAddress=secure@privateinternetaccess.com
Sun Jan  2 22:16:46 2022 VERIFY KU OK
Sun Jan  2 22:16:46 2022 Validating certificate extended key usage
Sun Jan  2 22:16:46 2022 ++ Certificate has EKU (str) TLS Web Server Authentication, expects TLS Web Server Authentication
Sun Jan  2 22:16:46 2022 VERIFY EKU OK
Sun Jan  2 22:16:46 2022 VERIFY OK: depth=0, C=US, ST=CA, L=LosAngeles, O=Private Internet Access, OU=Private Internet Access, CN=reykjavik401, name=reykjavik401
Sun Jan  2 22:16:46 2022 Control Channel: TLSv1.3, cipher TLSv1.3 TLS_AES_256_GCM_SHA384, 2048 bit RSA
Sun Jan  2 22:16:46 2022 [reykjavik401] Peer Connection Initiated with [AF_INET]45.133.193.41:1198
Sun Jan  2 22:16:47 2022 SENT CONTROL [reykjavik401]: 'PUSH_REQUEST' (status=1)
Sun Jan  2 22:16:47 2022 PUSH: Received control message: 'PUSH_REPLY,comp-lzo no,redirect-gateway def1,route-ipv6 2000::/3,dhcp-option DNS 10.0.0.243,route-gateway 10.7.112.1,topology subnet,ping 10,ping-restart 60,ifconfig 10.7.112.79 255.255.255.0,peer-id 1,cipher AES-128-GCM'
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: timers and/or timeouts modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: compression parms modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: --ifconfig/up options modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: route options modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: route-related options modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: --ip-win32 and/or --dhcp-option options modified
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: peer-id set
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: adjusting link_mtu to 1625
Sun Jan  2 22:16:47 2022 OPTIONS IMPORT: data channel crypto options modified
Sun Jan  2 22:16:47 2022 Data Channel: using negotiated cipher 'AES-128-GCM'
Sun Jan  2 22:16:47 2022 Outgoing Data Channel: Cipher 'AES-128-GCM' initialized with 128 bit key
Sun Jan  2 22:16:47 2022 Incoming Data Channel: Cipher 'AES-128-GCM' initialized with 128 bit key
Sun Jan  2 22:16:47 2022 ROUTE_GATEWAY 172.17.0.1/255.255.0.0 IFACE=eth0 HWADDR=02:42:ac:11:00:02
Sun Jan  2 22:16:47 2022 GDG6: remote_host_ipv6=n/a
Sun Jan  2 22:16:47 2022 ROUTE6: default_gateway=UNDEF
Sun Jan  2 22:16:47 2022 OpenVPN ROUTE6: OpenVPN needs a gateway parameter for a --route-ipv6 option and no default was specified by either --route-ipv6-gateway or --ifconfig-ipv6 options
Sun Jan  2 22:16:47 2022 OpenVPN ROUTE: failed to parse/resolve route for host/network: 2000::/3
Sun Jan  2 22:16:47 2022 TUN/TAP device tun0 opened
Sun Jan  2 22:16:47 2022 TUN/TAP TX queue length set to 100
Sun Jan  2 22:16:47 2022 /sbin/ip link set dev tun0 up mtu 1500
Sun Jan  2 22:16:47 2022 /sbin/ip addr add dev tun0 10.7.112.79/24 broadcast 10.7.112.255
Sun Jan  2 22:16:47 2022 /etc/openvpn/tunnelUp.sh tun0 1500 1553 10.7.112.79 255.255.255.0 init
Up script executed with tun0 1500 1553 10.7.112.79 255.255.255.0 init
Updating TRANSMISSION_BIND_ADDRESS_IPV4 to the ip of tun0 : 10.7.112.79
Using Combustion UI, overriding TRANSMISSION_WEB_HOME
Updating Transmission settings.json with values from env variables
Using existing settings.json for Transmission /data/transmission-home/settings.json
Overriding bind-address-ipv4 because TRANSMISSION_BIND_ADDRESS_IPV4 is set to 10.7.112.79
Overriding download-dir because TRANSMISSION_DOWNLOAD_DIR is set to /data/completed
Overriding incomplete-dir because TRANSMISSION_INCOMPLETE_DIR is set to /data/incomplete
Overriding rpc-port because TRANSMISSION_RPC_PORT is set to 9091
Overriding watch-dir because TRANSMISSION_WATCH_DIR is set to /data/watch
sed'ing True to true
Enforcing ownership on transmission config directories
Applying permissions to transmission config directories
Setting owner for transmission paths to 1000:1000
Setting permissions for download and incomplete directories
Mask: 002
Directories: 775
Files: 664
Setting permission for watch directory (775) and its files (664)

-------------------------------------
Transmission will run as
-------------------------------------
User name:   abc
User uid:    1000
User gid:    1000
-------------------------------------

STARTING TRANSMISSION
Provider PIA has a script for automatic port forwarding. Will run it now.
If you want to disable this, set environment variable DISABLE_PORT_UPDATER=true
Transmission startup script complete.
Privoxy: Starting
Privoxy: Using config file at /etc/privoxy/config
Privoxy: Setting port to 8118
Privoxy: Running as PID 105
Sun Jan  2 22:16:50 2022 /sbin/ip route add 45.133.193.41/32 via 172.17.0.1
Sun Jan  2 22:16:50 2022 /sbin/ip route add 0.0.0.0/1 via 10.7.112.1
Sun Jan  2 22:16:50 2022 /sbin/ip route add 128.0.0.0/1 via 10.7.112.1
Sun Jan  2 22:16:50 2022 WARNING: OpenVPN was configured to add an IPv6 route over tun0. However, no IPv6 has been configured for this interface, therefore the route installation may fail or may not work as expected.
Sun Jan  2 22:16:50 2022 Initialization Sequence Completed
Running functions for token based port fowarding
Reserved Port: 50741  Sun Jan  2 22:16:55 UTC 2022
transmission auth not required
waiting for transmission to become responsive
transmission became responsive
   1   100%    1.62 GB  Done         0.0     0.0    0.0  Verifying    XXX
Sum:           1.62 GB               0.0     0.0
setting transmission port to 50741
localhost:9091/transmission/rpc/ responded: "success"
Checking port...
Port is open: No
#######################
        SUCCESS        
#######################
Port: 50741
Expiration Sun Mar  6 10:18:50 UTC 2022
#######################
Entering infinite while loop
Every 15 minutes, check port status
Reserved Port: 50741  Sun Jan  2 22:32:05 UTC 2022
````

### Environment

```markdown
- OS: Debian 11.2
- Docker: 20.10.12, build e91ed57
```

### Anything else?

I noticed the problem a while ago, and simply stuck with version 3.6 as a workaround. From time to time I download the latest version, and until now the problem did not disappear. That's why I decided to file an issue now. Since there have been 3 releases since 3.6, I don't think the problem will be fixed in the dev branch.

**Comment by pkishino** [comment]
Have tried to replicate this here but I don’t see any issues using default UI.
Please post log output when using default UI, also, any browser console logs etc please.
Please use incognito or such mode with no extensions enabled.

**Comment by allefeld** [comment]
Using incognito mode indeed makes the problem go away.

To investigate, in regular mode I disabled extensions one-by-one, but even when all of them are disabled, the problem persists.

I also tried allowing all blocked functionality in Chrome's site settings (Pop-ups and redirects, Ads, Insecure content), with no effect.

Finally I found that if I access the UI using `localhost` instead of the configured host name, the problem goes away.

In Chrome, apparently that enables a number of site settings which are disabled for unsecured (`http`) hosts, which might be the explanation. Though going over them in detail I don't see any which the UI might need: Location, Camera, Microphone, Notifications, Background Sync, MIDI devices, Clipboard, Payment handlers, Augmented reality, Virtual reality, Your device use, Window placement, Fonts. I also wouldn't have thought that these disabled permissions become available in incognito mode.

In summary, I still don't know exactly what is happening, and why older versions of the UI don't seem to have this problem, but for me this is a viable workaround other than using an old version of the Docker image.

Thanks for your help!
